src/tokens.ts

```diff
@@ -406,6 +408,17 @@ export const TOKEN_SYMBOLS_MAP = {
     },
     coingeckoId: "weth",
   },
+  WGHO: {
+    name: "Gho Token",
+    symbol: "wGHO",
+    decimals: 18,
+    addresses: {
+      [CHAIN_IDs.LENS]: "0x6bDc36E20D267Ff0dd6097799f82e78907105e2F",
+      [CHAIN_IDs.MAINNET]: "0x40D16FC0246aD3160Ccc09B8D0D3A2cD28aE6C2f",
```

Comment:
Hm, I'm not 100% sure on this, because we have both GHO and wGHO as ERC20s on mainnet. This is currently the GHO address, but if we proceed with that then we'll have no reference for mainnet wGHO.